dask/dataframe/indexing.py

```diff
@@ -209,7 +210,7 @@ def _loc_element(self, iindexer, cindexer):
         return new_dd_object(graph, name, meta=meta, divisions=[iindexer, iindexer])
 
     def _get_partitions(self, keys):
-        if isinstance(keys, (list, np.ndarray)):
+        if isinstance(keys, (list, np.ndarray)) or is_arraylike(keys):
```

Comment:
The `np.ndarray`-type check should be covered by `is_arraylike`, so we should be able to remove it

```suggestion
        if isinstance(keys, list) or is_arraylike(keys):
```